lib/rubocop/sorbet/plugin.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "lint_roller"
+
+module RuboCop
+  module Sorbet
+    # A plugin that integrates RuboCop Sorbet with RuboCop's plugin system.
+    class Plugin < LintRoller::Plugin
+      def about
+        LintRoller::About.new(
+          name: 'rubocop-sorbet',
+          version: Version::STRING,
```

Comment:
Shouldn't this be pointing to `Sorbet::VERSION`?

Reply:
👍🏼